match/lib/match/importer.rb

```diff
@@ -9,10 +9,13 @@
 module Match
   class Importer
     def import_cert(params, cert_path: nil, p12_path: nil, profile_path: nil)
-      # Get and verify cert, p12 and profiles path
-      cert_path = ensure_valid_file_path(cert_path, "Certificate", ".cer")
-      p12_path = ensure_valid_file_path(p12_path, "Private key", ".p12")
-      profile_path = ensure_valid_file_path(profile_path, "Provisioning profile", ".mobileprovision or .provisionprofile", optional: true)
+      cert_path, p12_path, profile_path = resolve_inputs(params, cert_path, p12_path, profile_path)
+
+      if (cert_path.nil? && p12_path.nil? && profile_path.nil?) || (cert_path.nil? && !p12_path.nil?) || (!cert_path.nil? && p12_path.nil?)
```

Comment:
What about a little decomposition of this statement:
```suggestion
      all_files_are_nil = cert_path.nil? && p12_path.nil? && profile_path.nil?
      # Cert and private key must be provided together or not provided at all.
      both_cert_and_key_state_equal = cert_path.nil? == p12_path.nil?
      if all_files_are_nil || !both_cert_and_key_state_equal
```

PS. I have doubts about `both_cert_and_key_state_equal` naming.

Reply:
I'll take another look